src/main/java/com/project/bookapp/dto/book/BookDto.java

```diff
@@ -12,4 +13,5 @@ public class BookDto {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    private List<Long> categoryIds;
```

Comment:
Can you create a book with a category? Add this field to RequestDto and validate with NotEmpty